presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/InternalIcebergConnectorFactory.java

```diff
@@ -63,6 +64,7 @@ public static Connector createConnector(String catalogName, Map<String, String>
             Bootstrap app = new Bootstrap(
                     new EventModule(),
                     new MBeanModule(),
```

Comment:
It would be nice to have a test that validates metric names, e.g using iceberg query runnier with jmx connector.